delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/stream/StreamConverter.scala

```diff
@@ -45,12 +44,12 @@ object StreamConverter {
       .evalMap(publish)
       .unNoneTerminate
       .onFinalizeCase {
-        case ExitCase.Completed | ExitCase.Canceled => complete
-        case ExitCase.Error(e)                      => fail(e)
+        case ExitCase.Succeeded | ExitCase.Canceled => complete
```

Comment:
The fs2 finalize methods accept a `Resource.ExitCase`
A bit weird they renamed one `Outcome` but not this one